src/Core/AbstractConnection.php

```diff
@@ -140,7 +142,7 @@ public function getLogger()
      */
     protected function getHandler() : MessageHandlerInterface
     {
-        if ($this->handler instanceof \Closure) {
+        if (!$this->handler instanceof MessageHandlerInterface) {

```

Comment:
BC break ?

Reply:
No, it now allows callable as handler value. I just inversed the condition here. (you probably miss the `!`)